ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,117 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def get_drpc_name(namespace):
+    """
+    Get the DRPC Resource Name
+
+    Args:
+        namespace (str): Name of namespace
+
+    Returns:
+        str: DRPC resource name
+    """
+
+    drpc_obj = ocp.OCP(kind=constants.DRPC, namespace=namespace,).get()[
+        "items"
+    ][0]
+    return drpc_obj["metadata"]["name"]
+
+
+def get_drpolicy_name(namespace):
```

Comment:
Please correct me if i am wrong, But drpolicy is not namespace bound right ? why do we need namespace here, i assume its like PV which is not namespace bound.

Reply:
Yes no need for namespace will remove it